ansible/roles/fail2ban/tasks/main.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: Install EPEL repo
+  package:
+    name: epel-release
+
+- name: Install fail2ban packages
+  package:
+    name:
+      - fail2ban-server
+      - fail2ban-firewalld
+    state: present
+
+- name: Allow cluster subnet through filewalld
+  ansible.posix.firewalld:
```

Comment:
Personally, I think it would be cleaner if we had a separate role to configure the firewall, otherwise it can be tough to determine where all these rules came from. Could make the firewalld role run on hosts where fail2ban is enabled.

Reply:
I need to investigate this on a clean host. ISTR there was a problem that fail2ban configured firewalld too restrictively, which messed things up as soon as you started fail2ban, hence this approach. Putting to draft till resolved.

Reply:
@jovial as fail2ban installs firewalld it's hard to add a separate role for firewalld, as it would need to do:
- install and configure fail2ban (fail2ban role)
- configure and start firewalld (firewalld role)
- start fail2ban (either fail2ban role, or externally but that seems a bit odd)

However I do agree firewalld config needs to live in one place and be flexible, just hit the fact that OOD needs this configuring as well if you're running fail2ban and OOD on the login node.

What about this new approach in 08589c6?

Reply:
Couldn't you just make the firewalld role run first? Just seems like the role is doing two things.